package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "mobx": "^6.8.0",
+    "mobx-react-lite": "^3.4.3",
```

Comment:
so mobx is what we're using to create the stores (root store, playlist store, user store)

mobx-react-lite is a package that let's us integrate mobx into our react application. i'll write more comments about this below